lib/easypost/client.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require_relative 'services'
+require_relative 'client_configuration'
+require_relative 'http_client'
+require 'json'
+
+class EasyPost::Client
+  def initialize(api_key, api_base = nil, api_version = nil, read_timeout = 60, open_timeout = 30)
```

Comment:
issue: we will not allow users to choose the `api_version` when creating a client object since each version of a library will be directly tied to an API version. Please remove that parameter.

Reply:
We settled on that for the dynamic libs?

Reply:
Yes, many months ago. You won't find a parameter for this in those that are already finished with their rewrites.